Tests/TuistGeneratorTests/Generator/BuildPhaseGeneratorTests.swift

```diff
@@ -1124,7 +1124,7 @@ final class BuildPhaseGeneratorTests: TuistUnitTestCase {
                     showEnvVarsInLog: false,
                     basedOnDependencyAnalysis: false,
                     runForInstallBuildsOnly: true,
-                    dependencyFile: "/$(TEMP_DIR)/dependency.d"// testing dependency file
+                    dependencyFile: "/$(TEMP_DIR)/dependency.d" // testing dependency file
```

Comment:
```suggestion
                    dependencyFile: "/$(TEMP_DIR)/dependency.d"
```

Also, can you use a name different from the one in outputPaths, otherwise if we swap them by mistake the test wouldn't catch it? 🙏

Reply:
I made the change.
The test is actually failing because is adds `../` at the start of the path which i am not sure if is correct behaviour.

Reply:
might it be because you are using an absolute path, and the output is relative? 🤷

Reply:
Sorry, I was meaning that you should update the test to use the relative path, not viceversa 🙏

Reply:
i.e. it should be `dependencyFile: "$(TEMP_DIR)/dependency.d"` (without the leading `/`)